docker/scout/scripts/addresses.py

```diff
@@ -76,6 +82,28 @@
         "oracle": "0x058ec2Bf15011095a25670b618A129c043e2162E",
         "oracle_provider": "0x72dc16CFa95beB42aeebD2B10F22E55bD17Ce976",
     },
+    # --- ibBTC related ---
+    "peak_contracts": {
+        "badgerPeak": "0x41671BA1abcbA387b9b2B752c205e22e916BE6e3",
+        "byvWbtcPeak": "0x825218beD8BE0B30be39475755AceE0250C50627",
+    },
+    "interest_bearing_btc": {"token": "0xc4E15973E6fF2A35cC804c2CF9D2a1b817a8b40F"},
+}
+
+ADDRESSES_IBBTC = {
```

Comment:
Many of these addresses are already defined somewhere else, for example all the setts, and ren/wbtc and ibbtc 
 are tokens and the multisig is a wallet:)

It would be best if you either pulled them directly from those locations, and/or copied the addresses into the map referencing the other variable where they are stored to avoid someone changing something in one place and not realising it's duplicated.

Reply:
Keeping them separate is nice since the ibBTC event script doesn't need all the addresses from main and the mapping can be simpler. Updated `ADDRESSES_IBBTC` to reference addresses from `ADDRESSES_ETH`.